yt/frontends/enzo/tests/test_outputs.py

```diff
@@ -121,18 +119,18 @@ def test_moving7():
 def test_galaxy0030():
     ds = data_dir_load(g30)
     yield check_color_conservation(ds)
-    assert_equal(str(ds), "galaxy0030")
+    assert str(ds) == "galaxy0030"
     for test in big_patch_amr(ds, _fields):
         test_galaxy0030.__name__ = test.description
         yield test
-    assert_equal(ds.particle_type_counts, {"io": 1124453})
+    assert ds.particle_type_counts == {"io": 1124453}
```

Comment:
One thing I like about `assert_equal` is that it shows the actual and expected values in case of a failure.  Does running this in modern fixtures *also* provide that, or no?

Reply:
`pytest` monkey patches the `assert` statement to bridge the gap and make stuff like `assert_equal` unnecessary. *but* this particular test uses `yield` so it cannot be run with pytest until further effort is spent on migrating it. I am admittedly getting a little bit ahead of myself here. Should I revert this part ?

Reply:
I think it would be easier to review, and since we can't use the new monkeypatched version yet, let's keep it.

Reply:
done !